simGL/simGL.py

```diff
@@ -15,32 +16,30 @@ def extract_genotype_matrix(data):
     else:
         sys.exit("Incorrect data format")
 
-def depth_per_haplotype(rng, mean_depth, std_depth, n_ind):
-    if type(mean_depth) == int or type(mean_depth) == float:
+def depth_per_haplotype(rng, mean_depth, std_depth, n_hap, ploidy):
+    if (type(mean_depth) == int or type(mean_depth) == float) and mean_depth > 0.0:
```

Comment:
It's generally better to use `isinstance()` to check types, rather than comparing the types for equality. This is more flexible and robust in a variety of cases (e.g. it works with subclasses).
`if isinstance(mean_depth, (int, float)) and mean+depth > 0:`